Minor edits and a few questions

lasato-OSI opened the pull request:
This PR includes:

- Edits in response to Todd's comments
- Cleaned up terminology: ID -> Id; attribute -> metadata; metadatum -> metadata value
- Other minor editing and formatting
- Removed "Authorization" section from APIs
Please note: There are several questions. 

Thanks!

atangOSIsoft commented on Content_Portal/Documentation/Assets/Assets-API.md:
```diff
@@ -258,7 +258,7 @@ The newly created or updated `asset` object.
 If an asset type `Id` is specified for an asset, then the following is true:
 - The stream references name of an asset is set to null if the stream reference `Id` matches the stream reference `Id` of the asset type.
-- The name of a metadata value is set to null for those metadata values whose `Id` match the asset type metadata value `Id`s.
+- The name of a metadata value is set to null for those metadata values whose `Id` match the asset type metadata value `Id`s. <!-- This is confusing to me.Is this saying: If the asset and asset type have a metadata value with the same Id, then the name property on the asset is set to null -->
```
yup, how you phrased it is better.

lasato-OSI replied:
Done

atangOSIsoft commented on Content_Portal/Documentation/Assets/asset-default-shape.md:
```diff
@@ -16,10 +16,10 @@ Returns the default shape of an asset.
 When an asset references an asset type, the following rules explain how the values are derived for a given `Metadata` value on the default shape.
 
-- If a `Metadata` value with the same `Id` exists on the asset type and asset, the `Metadata` value properties, `Name`, `UOM`, and `SdsTypeCode`, of the asset type take precedence over the asset properties.
+- If a `Metadata` value with the same `Id` exists on the asset type and asset, the `Metadata` value properties, `Name`, `UOM`, and `SdsTypeCode`, of the asset type take precedence over the asset properties.<!-- Is Example 6 an example of this? -->
```
I would say no because example 6 is the names are different, and in this case, we don't even look at the asset when deriving the metadata value.

lasato-OSI replied:
okay

atangOSIsoft commented on Content_Portal/Documentation/Assets/asset-default-shape.md:
```diff
@@ -33,25 +33,26 @@ The examples in the following table illustrate the default shape metadata value
-The following example illustrates the default shape reference rules. If an asset references an asset type, the the following rules apply for a given **Reference**:
-
+The following example illustrates the default shape reference rules. If an asset references an asset type, the the following rules apply for a given `StreamReferences`:
+<!-- I changed **Reference** to `StreamReferences`. Is that correct? -->
```
I would leave as a generic references since asset types have "TypeReferences" while assets have "StreamReferences".

lasato-OSI replied:
Okay, reverted back to "reference."

atangOSIsoft commented on Content_Portal/Documentation/Assets/asset-status-mapping.md:
```diff
@@ -14,10 +14,10 @@ The following table lists the most common fields in a status mapping.
+| Description         | String                   | Required  | No          | `Description` of the status mapping.                         |
 | StreamReferenceId   | String                   | Required  | No          | `Id` for the asset's StreamReference property. The stream reference must exist before the status mapping can be created. |
-| StreamPropertyId    | String                   | Required  | No          | SDS stream property that status uses for calculations. It must be present on the StreamId property on the asset StreamReference. |
+| StreamPropertyId    | String                   | Required  | No          | SDS stream property that status uses for calculations. It must be present on the StreamId property on the asset StreamReference.  The mapped measurement must be a numeric enumeration, character, or string type. |
```
This is more accurate to say "stream property type"

lasato-OSI replied:
Sorry, do you mean it should say: SDS stream property type that status uses for calculations. ....

atangOSIsoft replied:
Apologize for not being clear, GIT Hub doesn't highlight the exact section.

I mean mapped measurement is not totally accurate since status mapping doesn't necessarily use the measurement mappings, hence, the sentence " The mapped measurement must be a numeric enumeration, character, or string type." should be " The Sds stream property must be a numeric, enumeration, character, or string type."

lasato-OSI replied:
Okay, thanks. Done.

atangOSIsoft commented on Content_Portal/Documentation/Assets/asset-status-mapping.md:
````diff
@@ -64,7 +64,7 @@ The following is an example of a status mapping.
 ```
 
-The asset or asset type's  StreamReferences field has an Id property. To assign a status mapping to an asset or asset type, the value assigned to the Id property must match the StreamReferenceId of the status mapping object. Using the status mapping example above, AssetStreamReferenceId is assigned to the asset in the following example. 
+The asset or asset type's  StreamReferences field has an Id property. To assign a status mapping to an asset or asset type, the value assigned to the Id property must match the StreamReferenceId of the status mapping object. Using the status mapping example above, AssetStreamReferenceId is assigned to the asset in the following example. <!-- Should this be AssetStreamReferenceId1? -->
````
Yup! Thanks!

lasato-OSI replied:
Done

lasato-OSI commented on Content_Portal/Documentation/Assets/Asset-Type.md:
```diff
@@ -230,9 +211,9 @@ Create or update an asset type with a specified `Id`.
+- Deleting the type reference on the asset type deletes the measurements on the default shape of referenced assets​.
 - Renaming a metadata value on the asset type renames the metadata value on the default shape of referenced assets​.
-- Renaming the stream reference name on the asset type on default shape of referenced assets
+- Renaming the stream reference name on the asset type on default shape of referenced assets<!-- Is this sentence complete? -->
```
Done

lasato-OSI commented:
Merging the following changes:
Edits in response to Todd's comments
Cleaned up terminology: ID -> Id; attribute -> metadata; metadatum -> metadata value
Other minor editing and formatting
Removed "Authorization" section from APIs
Clarification of wording.
